docs/baking.md

```diff
@@ -0,0 +1,175 @@
+package.
+
+However, by default, these services will start to bootstrap the node from scratch,
+which will take a significant amount of time.
+In order to avoid this, here we suggest bootstrapping from a snapshot instead.
+
+### Setting up node service
+
+`tezos-node-<network>.service` has `/var/lib/tezos/node-<network>` as a data directory
+and `http://localhost:8732` as its RPC address by default.
+
+In case you want to use different data directory or RPC address,
+you should update the service configuration. To edit the service configuration run
+```
+sudo systemctl edit --full tezos-node-<network>.service
+```
+
+### Bootstrapping the node
+
+In order to run a baker locally, you'll need a fully-synced local `tezos-node`.
+
+The fastest way to bootstrap the node is to import a snapshot.
+Snapshots can be downloaded from the following websites:
+* https://snapshots.tulip.tools/#/
+* https://snapshots-tezos.giganode.io/
+* https://xtz-shots.io/
+
+All commands within the service are run under the `tezos` user.
+
+Download the snapshot for desired network. We recommend to use rolling snapshot, this is
+the smallest and the fastest mode that is sufficient for baking (you can read more about other
+`tezos-node` history modes [here](https://tezos.gitlab.io/user/history_modes.html#history-modes)).
+
+In order to import the snapshot run the following command:
+```
+sudo -u tezos tezos-node snapshot import --data-dir /var/lib/tezos/node-<network>/ <path to the snapshot file>
+```
+
+#### Troubleshooting node snapshot import
+
+Snapshot import requires that the `tezos-node` data directory doesn't contain the `context` and `store` folders.
+Remove them in case you're getting such error about their presence.
+
+In case you're getting an error similar to
+```
+tezos-node: Error:
+              Invalid block BLjutMj47caB
+                Failed to validate the economic-protocol content of the block: Error:
+                                                                                Invalid signature for block BLjutMj47caB. Expected: tz1Na5QB98cDA.
+```
+
+You should init/update config in the data directory to match the desired network
+before importing the snapshot. To do that run one of the following commands
+depending on whether the config in the given data directory was initialized previously:
+```
+sudo -u tezos tezos-node config init --data-dir /var/lib/tezos/node-<network> --network <network>
+#or
+sudo -u tezos tezos-node config update --data-dir /var/lib/tezos/node-<network> --network <network>
+```
+
+### Starting the node
+
+After the snapshot import, you can finally start the node by running
+```
+sudo systemctl start tezos-node-<network>.service
+```
+
+Note that even after the snapshot import the node can still be out of sync, it may require
+some additional time to completely bootstrap. In order to check whether the node is bootstrapped,
+you can use `tezos-client`:
+```
+tezos-client --endpoint <node-rpc-address> bootstrapped
+```
```

Comment:
Actually even better, if the tezos client default config had the right values this could be just replaced with:
```
sudo -u tezos tezos-client bootstrapped
```